demo/getset.html

```diff
@@ -0,0 +1,118 @@
+                                "color": "green",
+                                "id": 11
+                            },
+                            {
+                                "name": "Truck",
+                                "color": "orange",
+                                "id": 12
+                            },
+                        ],
+                        "allowed_modes": ["bbox", "polygon", "contour", "bbox3"],
+                        "resume_from": null,
+                        "task_meta": null,
+                        "annotation_meta": null,
+                        "read_only": false,
+                        "inactive_opacity": 0.6
+                    },
+                    "frame_review": {
+                        "display_name": "Frame Review",
+                        "classes": [
+                            {
+                                "name": "Blurry",
+                                "color": "gray",
+                                "id": 20
+                            },
+                            {
+                                "name": "Occluded",
+                                "color": "red",
+                                "id": 21
+                            }
+                        ],
+                        "allowed_modes": ["whole-image", "global"],
+                        "resume_from": null,
+                        "task_meta": null,
+                        "annotation_meta": null,
+                        "read_only": false
+                    }
+                };
+
+                // Initial ULabel configuration
+                let ulabel = new ULabel(
+                    "container",
+                    [
+                        "https://ulabel.s3.us-east-2.amazonaws.com/cs-demo-0.png", 
+                        "https://ulabel.s3.us-east-2.amazonaws.com/cs-demo-1.png", 
+                        "https://ulabel.s3.us-east-2.amazonaws.com/cs-demo-2.png"
+                    ],
+                    "DemoUser",
+                    on_submit,
+                    subtasks,
+                    null,
+                    null,
+                    1,
+                    null
+                );
+                // Wait for ULabel instance to finish initialization
+                ulabel.init(function() {
+                    // ULabel is now ready for use
+                    document.body.onkeydown = function(keypress_event) {
+                        if (keypress_event.key == "g") {
+                            let anns = ulabel.get_annotations("car_detection");
+                            for (let i = 0; i < anns.length; i++) {
+                                for (let j = 0; j < anns[i]["spatial_payload"].length; j++) {
+                                    anns[i]["spatial_payload"][j][0] += 10;
+                                }
+                            }
+                            ulabel.set_annotations(anns, "car_detection");
```

Comment:
If you run this while in the middle of making an annotation, it will never let you finish making it, and doesn't let you switch to other tools.

Reply:
Specifically if it's pressed between mousedown and mouseup